packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -923,12 +921,10 @@ function finishConcurrentRender(root, exitStatus, lanes) {
     case RootSuspendedWithDelay: {
       markRootSuspended(root, lanes);
 
-      if (workInProgressRootLatestSuspenseTimeout !== NoTimestamp) {
+      if (includesOnlyTransitions(lanes)) {
```

Comment:
Just confirming, for the logic in the next block (GitHub won't let me comment there), removing the `eventTime` doesn't impact the JND logic that's using `root.eventTimes` right?

Reply:
Right, that was decoupled as part of https://github.com/facebook/react/pull/19387